logger.js

```diff
@@ -46,13 +46,22 @@ function pinoLogger (opts, stream) {
     }, 'request completed')
   }
 
+  function onReqAborted () {
+    var res = this.res
+    res.statusCode = 408
+    onResFinished.call(res, new Error('Aborted'))
+  }
+
   function loggingMiddleware (req, res, next) {
     req.id = genReqId(req)
     req.log = res.log = logger.child({req: req})
     res.startTime = Date.now()
+    if (!req.res) { req.res = res }
```

Comment:
I don't get why we have to check for `req.res` before setting it.
IMO we can remove that. Just use `req.res = res` directly.